cvat/apps/engine/models.py

```diff
@@ -994,7 +994,7 @@ class Issue(models.Model):
     assignee = models.ForeignKey(User, null=True, blank=True, related_name='+',
         on_delete=models.SET_NULL)
     created_date = models.DateTimeField(auto_now_add=True)
-    updated_date = models.DateTimeField(null=True, blank=True)
+    updated_date = models.DateTimeField(null=True, blank=True, auto_now=True)
```

Comment:
So, in this case the `updated_date` fields will be not null only for issues, that will be updated after that fix. Should we add  `RunPython` migration to define default values for all issue objects? (to prevent data inconsistent)

Reply:
I don't really think it is important

Reply:
IMHO, I don't think that it should be possible at all to have a null `updated date` field.